pkg/deploy/dashboard/dashboard_test.go

```diff
@@ -60,21 +60,21 @@ func TestDashboardAll(t *testing.T) {
 
 	// check service
 	service := &corev1.Service{}
-	err = cli.Get(context.TODO(), types.NamespacedName{Name: "dashboard", Namespace: "eclipse-che"}, service)
+	err = cli.Get(context.TODO(), types.NamespacedName{Name: "che-dashboard", Namespace: "eclipse-che"}, service)
```

Comment:
I think you can use constant DashboardComponent instead of "che-dashboard". But up to you.

Reply:
Make sense to me.